### PR TITLE
if a url has a port include it when calling buildUrl

### PR DESCRIPTION
**Pull request by scashin133**
This should resolve https://github.com/imgix/imgix.js/issues/44

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -1830,7 +1830,11 @@ imgix.parseUrl = function (url) {
 };
 
 imgix.buildUrl = function (parsed) {
-  var result = parsed.protocol + '://' + parsed.host + parsed.pathname;
+  var result = parsed.protocol + '://' + parsed.host;
+  if (parsed.port !== '80' && parsed.port !== '443') {
```
Comment (jayeb) [inline src/core.js]: Is there ever a situation where `parsed.port` will be `null` or `undefined`?

Comment (scashin133) [inline src/core.js]: I don't think it will but to be safe I'll add an additional check

**Comment by jayeb** [comment]
@scashin133 Can you kill the `dist/*` files from this branch? They get built automatically when you run tests. I'll rebuild them when it's time to cut a new version for release.

**Comment by jayeb** [comment]
Closing this PR in favor of #62.

**Pull request closed without merging**